android/src/main/java/tech/bam/RNBatchPush/RNBatchPushModule.java

```diff
@@ -74,6 +76,24 @@ public void logoutUser() {
       .save();
   }
 
+  @ReactMethod
+  public void setAttribute(String key, String value) {
+    Batch.User.editor()
+      .setAttribute(key, value)
+      .save();
+  }
+
+  @ReactMethod
+  public void trackLocation(ReadableMap locationMap) {
+    Location location = new Location("reactNative");
+    location.setAccuracy((float) locationMap.getDouble("accuracy"));
```

Comment:
In the docs there are no mentions of accuracy, altitude and speed, would you not rather not set the accuracy, altitude and speed of the Location? So that's the same as iOS

Reply:
Batch uses Android `Location` class. As a result, I am not sure which parameters Batch use. So I decided to construct the best Android `Location` instance with all details I have from the returned value of `RN.geolocation.getCurrentPosition()`.
For example, in the Batch Dashboard, I see these values:
```json
  "geo_accuracy": 17.65,
  "geo_last_update": "2018-01-09T17:00:06",
  "geo_latitude": 48.8827743,
  "geo_longitude": 2.3222816
```

Would you like to remove these lines anyway ?

Reply:
If it doesn't crash if you doesn't provide any accuracy/altitude/speed to the location javascript object, it's fine 👍

Reply:
If not, maybe update the doc to say that these keys are mandatory on Android